sage-ios/SAGE/SAGE/BrowseMentorsViewController.swift

```diff
@@ -44,10 +46,25 @@ class BrowseMentorsViewController: UITableViewController {
         self.alphabetizeAndLoad(users)
     }
     
+    func setSemesterTitle(semesterID: String) {
+        SemesterOperations.getSemester(semesterID as String, completion: { (semester) -> Void in
+            self.titleView.setSubtitle(semester.displayText())
+            }) { (errorMessage) -> Void in
+                self.showErrorAndSetMessage(errorMessage)
```

Comment:
probably dont wanna show any error message and just set the subtitle to like "Semester " or somethin